### ISSUE TITLE
Upgrade system to 10.x series LTS for Node.js

### ISSUE THREAD

**Issue by kltm** [geneontology/noctua#625]
Closing this ticket would involve upgrading our target platform to the latest Node.js LTS. Most basically, this means running all compilations and tests for 10.x LTS on our current 18.04 bionic target, as well as functional testing.

Note that this involves upgrading the target JS platform for all GO software, not just Noctua, and needs to be done carefully.

Also note #364

**Comment by vanaukenk** [comment]
@kltm - would you like to leave this one open?

**Comment by kltm** [comment]
Yes, I think that needs to remain open--we need to generally update our codebase versions.

**Comment by pkalita-lbl** [comment]
Just noting that 10.x is already EOL. When we get to this, find the latest LTS version here: https://github.com/nodejs/release#release-schedule
